docs/source/getting_started/authoring_your_workflow.md

```diff
@@ -18,91 +18,85 @@ The tutorial aims to be an extension to the quickstart to help you better unders
 
 ## 1: Initialize Workflow Directory
 Bootstrap a new workflow directory by running `latch init` from the command line.
+
+```shell-session
+$ latch init covid-wf -t subprocess
+Created a latch workflow in `covid-wf`
+Run
+        $ latch register covid-wf
+To register the workflow with console.latch.bio.
 ```
-latch init covid-wf
-```
-Output:
-```
-Created a latch workflow called covid-wf.
-```
+
 File Tree:
-```
+```shell-session
 covid-wf
-├── Dockerfile
+├── LICENSE
+├── README.md
+├── bowtie2
+│   ├── bowtie2
+│   └── ...
 ├── reference
 │   ├── wuhan.1.bt2
-│   ├── wuhan.2.bt2
-│   ├── wuhan.3.bt2
-│   ├── wuhan.4.bt2
-│   ├── wuhan.fasta
-│   ├── wuhan.rev.1.bt2
-│   └── wuhan.rev.2.bt2
+│   └── ...
+├── system-requirements.txt
 ├── version
 └── wf
-    └── __init__.py
+    ├── __init__.py
+    ├── assemble.py
+    └── sort.py
 ```
+
 Once your boilerplate workflow has been created successfully, you should see a folder called `covid-wf`.
 
 ## 2. Writing your First Task
 
-A task is a pure Python function that takes in inputs and returns outputs. Here, we are writing a task that takes in two sequencing reads and returns an assembled SAM file.
+A task is a Python function that takes in inputs and returns outputs to the Latch platform or to another task. In the `assemble.py` file inside the `covid-wf/wf` directory, we have a task which ingests two sequencing reads and outputs an assembled SAM file.
+
+### How to work with LatchFiles and LatchDirs
+
+`LatchFile` and `LatchDir` are types built into the Latch SDK which allow users to use files and directories on Latch as inputs and outputs to workflows. They point to remote file locations in our user interface on [Latch Console](https://console.latch.bio/data) and implement the necessary operations to ensure data is pulled into the task environment and saved back into your data tab.
```

Comment:
add :class: references to `LatchFile` and `LatchDir`

Reply:
"use files and directories on Latch" is confusing
suggestion: "use files and directories stored remotely on Latch"

Reply:
would prefer "the Latch Console" over "Latch Console"

Reply:
"pulled into" is silly
suggestion: "available in"

"saved back" is silly
suggestion: "outputs are uploaded to your Latch account"